webclient/arch_z/views.py

```diff
@@ -845,12 +846,30 @@ def vratit(request, ident_cely):
             {"redirect": az.get_absolute_url()},
             status=403,
         )
+    DokumentFormSet = formset_factory(VratitFormDokument, extra=0)
     if request.method == "POST":
-        form = VratitForm(request.POST)
-        if form.is_valid():
+        form = VratitFormAkce(request.POST, ident=az.ident_cely)
+        formset = DokumentFormSet(request.POST)
```

Comment:
Není mi úplně jasné, kde se formset omezí jen na dokumenty připojené k AZ. Prosím o vysvětlení, asi to ale bude jen mé špatné čtení kódu.